pyblish_qml/models.py

```diff
@@ -384,7 +384,11 @@ def add_instance(self, instance):
         item["hasCompatible"] = True
 
         # Visualised in Perspective
-        item["familiesConcatenated"] = instance["data"].get("family", "")
```

Comment:
It's for verbosity and readability only, so it could live in the same block. But I wouldn't go for a function, I'm mostly spitballing. So long as it reads better and is less verbose, it's ok.